indico/modules/events/templates/emails/move_request_closure.txt

```diff
@@ -0,0 +1,33 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    Event move request update
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    {%- filter dedent -%}
+        {% if accept %}
+            The following events have been moved:
+        {% else %}
+            Move requests of the following events have been rejected:
+        {% endif %}
+
+        {% filter underline %}Events{% endfilter %}
```

Comment:
IMHO the underlined "Events" header looks a bit strange. I'd get rid of it and put `- ` in front of each event entry.

```
The following events have been moved:

- MOAR EVENTS (https://.../event/1056189/) 
- mod test (https://.../event/1056185/) 
```

vs

```
The following events have been moved:


Events
------
MOAR EVENTS (https://.../event/1056189/) 
mod test (https://.../event/1056185/) 
```